public/js/index.js

```diff
@@ -209,6 +209,10 @@ function initInstall() {
         return;
     }
 
+    $('#db_host').val("127.0.0.1:3306");
+    $('#db_user').val("root");
```

Comment:
I see,"just because it's there" is not good and PostgreSQL & MSSQL is other,I updated.